Add kubeflow-behind-proxy.md - see issue #2137 (#2147)
kubeflow/website: content/en/docs/other-guides/kubeflow-behind-proxy.md

+++
title = "Deploying Kubeflow behind a proxy server"

> description = "Instructions to install and configure Kubeflow behind a proxy server"

Reviewer: Just a small suggestion: 

- Use "instructions for [doing]" instead of "instructions to [do]" in the description

```suggestion
description = "Instructions for installing and configuring Kubeflow behind a proxy server"
```

🤷‍♀️ 

It's maybe more grammatically correct + based on the following:

<img width="900" alt="image" src="https://user-images.githubusercontent.com/19637339/98407289-4ff35700-2067-11eb-9fba-efbcb4f1d10b.png">

Reviewer: @RFMVasconcelos Hey, let us know if there'll be any updates. Just following up 😃 Have a great weekend

weight = 10

+++

## Install Kubeflow behind a proxy server

This guide demonstrates how to install and configure Kubeflow behind a proxy server.

**Note**: These instructions assume that your Docker setup has a [configured
proxy service](https://docs.docker.com/network/proxy/) on each Kubernetes node.

1. Create an alias for the proxy:

> ```

Reviewer: Here and everywhere else in this doc where you've got a code block, let's add `shell` like so:
```suggestion
```shell
```

alias https='https_proxy=https://{SERVER}:{PORT}'
```

where `{SERVER}` is the IP address of your proxy server and `{PORT}` is the port number.

2. Download the latest [kfctl](https://github.com/kubeflow/kfctl) — the control
   pane for deploying and managing Kubeflow from the [Kubeflow
   releases](https://github.com/kubeflow/kfctl/releases/) page. For example, run
   the command below to download the zipped kfctl TAR file for Kubeflow v1.1 and
   extract it to `/usr/local/bin`:

> ```

Reviewer: ```suggestion
```shell
```

https wget https://github.com/kubeflow/kfctl/releases/download/v1.1.0/kfctl_v1.1.0-0-g9a3621e_linux.tar.gz extract to /usr/local/bin
```

3. To install Kubeflow on an existing cluster, you need to download the the
   latest configuration YAML file. For example, for Kubeflow v1.1 it is:

```
https wget https://raw.githubusercontent.com/kubeflow/manifests/v1.1-branch/kfdef/kfctl_istio_dex.v1.1.0.yaml
```

4. Download the KfDef manifest zipped TAR file. For example, for Kubeflow v1.1
   you should run the following command:

> ```

Reviewer: ```suggestion
```shell
```

https wget https://github.com/kubeflow/manifests/archive/v1.1-branch.tar.gz
```

5. Open the YAML configuration (zipped) file and under `- name: manifests`
   change the old URI (such as
   `https://github.com/kubeflow/manifests/archive/v1.1-branch.tar.gz`) to the

>    location of the new one, as follows:

Reviewer: ```suggestion
   location of the new one as follows:
```

> ```

Reviewer: ```suggestion
```shell
```
